R/naiveDE.R

```diff
@@ -4,28 +4,33 @@
 #' This function is a wrapper for `stabilize` from the
 #' [NaiveDE](https://github.com/Teichlab/NaiveDE) Python package.
 #'
-#' @param counts `matrix` or `data.frame` with expression values for samples in
+#' @param counts `matrix` with expression values for samples in
 #' columns and genes in rows.
 #'
 #' @return `matrix` of variance stabilized counts.
 #'
 #' @examples
+#' \dontrun{
```

Comment:
Any reason for not running the example? Note that for BioC, at least 80% of exported functions should have runnable examples: https://www.bioconductor.org/developers/package-guidelines/#manpages

Reply:
Fixed !